### ISSUE TITLE
convert endpoint response : "message" instead of "error"

### ISSUE THREAD

**Issue by tcharlss** [playht/text-to-speech-api#1]
Hi, it seems the key explaining the error in the convert endpoint response is now called "message" instead of "error":

```
{
  "status": "transcriping" | "error",
  "transcriptionId": string,
  "error": string // Optional
}
```

should be:

```
{
  "status": "transcriping" | "error",
  "transcriptionId": string,
  "message": string // Optional
}
```

**Comment by HazemAM** [comment]
Hello @tcharlss. Thank you very much for the report!

We apologize about the delay. This will be fixed in a release within the following hours.

Please note that `error` is the intended key here. So the fix will unify the key to `error` instead of `message` being used in some places.

Have a great day/evening!

**Comment by HazemAM** [comment]
Fixed in `v1.0.1`.
